aptos-move/move-examples/vote_lockup/sources/package_manager.move

```diff
@@ -0,0 +1,66 @@
+module vote_lockup::package_manager {
+    use aptos_framework::account::{Self, SignerCapability};
+    use aptos_framework::resource_account;
+    use aptos_std::smart_table::{Self, SmartTable};
+    use std::string::String;
+
+    friend vote_lockup::vote;
+    friend vote_lockup::voting_token;
+
+    /// Stores permission config such as SignerCapability for controlling the resource account.
+    struct PermissionConfig has key {
+        /// Required to obtain the resource account signer.
+        signer_cap: SignerCapability,
+        /// Track the addresses created by the modules in this package.
```

Comment:
Let me figure out in a follow up PR how to use a common package_manager instead of copy pasting it for each example. I'll go through and update the documentation more thoroughly as well